resources/lang/de.json

```diff
@@ -895,5 +895,7 @@
 	"Delete this reply": "Kommentar löschen",
 	"Post and all replies and comments in this post successfully deleted.": "Der Beitrag und alle Antworten und Kommentare wurden erfolgreich gelöscht",
 	"Post successfully restored.": "Beitrag erfolgreich wiederhergestellt.",
-	"Post successfully updated.": "Beitrag erfolgreich aktualisiert."
```

Comment:
Why did you delete this line, @milack68? Don't we need this anymore?

Reply:
You are right. I will add this line once again.